modules/migration-compatibility-guidelines.adoc

```diff
@@ -20,22 +20,25 @@ modern operator:: The {mtc-short} Operator designed for modern platforms.
 control cluster:: The cluster that runs the {mtc-short} controller and GUI.
 remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
 
-
-[cols="1,2,2", options="header"]
 .{mtc-short} compatibility: Migrating from a legacy platform
 |===
-||{product-title} 4.5 or earlier |{product-title} 4.6 or later
-|Stable {mtc-short} version a|{mtc-short} {mtc-version}._z_
+| |{product-title} 3.11 |{product-title} 4.0 to 4.5 |{product-title} 4.6 or later
+
+|Stable {mtc-short} version
+|{mtc-short} {mtc-version}v.1.7
```

Comment:
in the preview, it displays like this -> MTC 1.8v.1.7

MTC 1.8 is not supported for OCP 3.x, so it should not be present here.

Reply:
@stillalearner 
Thanks for noticing that. I removed it.